icons/square-dashed-bottom.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3 17V5a2 2 0 0 1 2-2h14a2 2 0 0 1 2 2v12" />
+  <path d="M4 21h1" />
+  <path d="M9 21h1" />
+  <path d="M14 21h1" />
+  <path d="M19 21h1" />
+</svg>
```

Comment:
What about something like this ![icon (48)](https://github.com/lucide-icons/lucide/assets/25524993/b31ff1e2-9b8d-4a31-a2b3-35a27250dffb)?

```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M5 21a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2" />
  <path d="M9 21h1" />
  <path d="M14 21h1" />
</svg>
```